js/component/Sessions.vue

```diff
@@ -40,8 +40,13 @@
             </div>
           </div>
           <div>
-            <div class="text-thawani md:hidden">{{ $t("order_id") }}</div>
-            {{ session.client_reference_id.trim() }}
+            <a
+              :href="`./post.php?post=${session.metadata.order_id}&action=edit`"
+              target="_blank"
+              class="bg-blue-100 text-blue-500 hover:bg-blue-900 hover:text-white rounded inline-block p-1 w-full lg:w-4/5 shadow text-center uppercase"
```

Comment:
```suggestion
              class="text-blue-500 hover:text-blue-800"
```

I know there is no design/prototype for the dashboard. 
but here what is the design should be https://play.tailwindcss.com/iZUbClgDIK

Reply:
@HajerSubhi if you don't have time to work on this let me know so I can take over this

Reply:
> @HajerSubhi if you don't have time to work on this let me know so I can take over this

No no, it will be done before the end of today.